playbooks/zuul-log-server.yaml

```diff
@@ -23,6 +23,22 @@
         mode: 0755
         state: directory
 
+    - name: Set host private ecdsa key
+      when: config_host_ecdsa_key|default(true)|bool
+      copy:
+        content: "{{ known_host_key_priv }}"
+        dest: /etc/ssh/ssh_host_ecdsa_key
+        mode: 0600
```

Comment:
Double check: is this for `root` user, not `zuul` user?

Reply:
all the files under `/etc/ssh/` are root:root owned